src/helpers.js

```diff
@@ -0,0 +1,3 @@
+export const amount = length => Array.from({ length }, (_, i) => i);
```

Comment:
`helper` functions/classes are seldom a good idea, in this case also. COnsider just using https://underscorejs.org

Reply:
Could you explain why? I saw this solution couple of times in other projects. I also wanted to write such a function myself for learning purposes.

Reply:
You can also build your own computer for learning purposes but this is not what this is about, is it?